sources/packages/web/src/views/aest/StudentDetails.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <h5 class="text-muted">
+    <a @click="goBack()">
+      <v-icon left> mdi-arrow-left </v-icon> Back to Student Search</a
+    >
+  </h5>
+  <PDStatus :pdStatus="studentDetail.pdStatus" />
+  <full-page-container>
```

Comment:
same here, I thought ` <full-page-container` is a root container

Reply:
The _full-page-container_ is the white part of the page where the content leaves.
Things like headers are not supposed to leave on it as it is right now, **but we can extend the component to have one slot for header and one for the "body".**

Reply:
I did not create any common component for header portion.